api/api-common/src/main/kotlin/io/qalipsis/api/meters/steps/CounterMeterStepSpecification.kt

```diff
@@ -41,7 +41,7 @@ interface CounterMeterStepSpecification<INPUT> :
     /**
      * Allows specification of failure conditions on the count meter.
      */
-    fun shouldFailWhen(block: CounterFailureConditionSpec.() -> Unit): ConfigurableStepSpecification<INPUT, INPUT, *>
+    fun shouldMatch(block: CounterFailureConditionSpec.() -> Unit): ConfigurableStepSpecification<INPUT, INPUT, *>
```

Comment:
I understand the name change, but `shouldMatch` sounds a bit ambiguous to me because it sounds like we are matching a Regex pattern.  
What do you think about `shouldSatisfy`, `shouldMeet`, or `satisfies`?